cfme/intelligence/reports/reports.py

```diff
@@ -367,6 +367,9 @@ def __init__(self, path_to_report, datetime, candu=False, appliance=None):
         self.candu = candu
         self.datetime_in_tree = version.pick({"5.6": self.datetime,
                         "5.7": parsetime.from_american_with_utc(self.datetime).to_iso_with_utc()})
+        self.datetime_in_tree_delete = version.pick({"5.6": self.datetime,
+                        "5.7": parsetime.from_american_with_utc(
+                            self.datetime).to_american_with_utc()})
```

Comment:
Why is it necessary to convert `from_american_with_utc` -> `to_american_with_utc`? What difference does it make to  `self.datetime`?
Why does the datetime differs for delete? It's the same tree, right?